tests/test_generate_fasta.py

```diff
@@ -5,33 +5,121 @@
 from subprocess import call
 from filecmp import cmp
 import py_compile
+from pvacseq.lib import generate_fasta
 
-class GenerateFastaTests(unittest.TestCase):
-    def setUp(self):
-        self.python = sys.executable
+class GenerateVariantSequences(unittest.TestCase):
```

Comment:
Ah yes, that was an oversight when I resolved merge conflicts after rebasing. It's now fixed.